andela/activo-web
# #159569195 Users can change default center

**What does this PR do?**

This PR adds functionality to allow a user to change their default center from the profile dropdown

**Description of Task to be completed?**

Given that I am a user logged into Activo, I should be able to navigate to my settings page and change my default center

**How should this be manually tested?**

After cloning and running the branch click on the drop-down icon on the far right of the navigation bar, a dropdown menu will appear. Click the `change` button and a modal with a list of centers will appear. Select one center and click save changes.

**Any background context you want to provide?**

- The endpoint to persist this feature in the backend is not ready, therefore I had to design a module user that will contain the current logged in user details. I would like to ask for feedback on this module in order to improve it.

**What are the relevant pivotal tracker stories?**

[#159569195 As a user, I should be able to set my default center](https://www.pivotaltracker.com/story/show/159569195)

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2018-08-13 at 10 42 05" src="https://user-images.githubusercontent.com/17454824/44018822-a8c77204-9ee5-11e8-9166-c1f6d20e64e0.png">

<img width="1440" alt="screen shot 2018-08-13 at 10 44 06" src="https://user-images.githubusercontent.com/17454824/44018889-df4c81e8-9ee5-11e8-96cc-d970599b165a.png">

**Questions:**

Suggestion on how the user information that is not provided in the token be stored in the state?
